seqr/views/apis/project_api.py

```diff
@@ -221,11 +233,9 @@ def project_overview(request, project_guid):
 
     sample_models = Sample.objects.filter(individual__family__project=project)
 
-    active_samples = sample_models.filter(is_active=True)
-    first_loaded_samples = sample_models.order_by('individual__family', 'loaded_date').distinct('individual__family')
-    samples_by_guid = {}
-    for samples in [active_samples, first_loaded_samples]:
-        samples_by_guid.update({s['sampleGuid']: s for s in get_json_for_samples(samples, project_guid=project_guid)})
+    first_loaded_samples = sample_models.order_by('individual__family', 'loaded_date').distinct('individual__family').values_list('id', flat=True)
```

Comment:
this still assesses to 2 DB queries, but the query that computed `first_loaded_samples` and pulled all the json fields out for it was actually relatively inefficient due to the underlying aggregation and it was cheaper to do the aggegration as one query and then pull out all the json fields for all the relevant samples as a separate query